examples/ini.js

```diff
@@ -0,0 +1,47 @@
+const fs = require("fs");
+const ini = require("ini");
+const utils = require('../utils');
+const path = require('path');
+const Store = require('../');
+
+// Will be called in context of store
+function writeFile() {
+  console.log("writeFile");
+  utils.mkdir(path.dirname(this.path));
+  fs.writeFileSync(this.path, ini.stringify(this.data), { mode: 0o0600 });
+}
+
+// Will be called in context of store
+function readParseFile() {
+  let data;
+  
+  console.log("readParseFile");
+  try
+  {
+    data = fs.readFileSync(this.path, "utf-8");
+  }
+  catch (e)
+  {
+    console.log(`readParseFile error; starting with empty data`);
```

Comment:
Please use single quotes unless there's something that will be used in the template string:

```suggestion
    console.log('readParseFile error; starting with empty data');
```